unittests/FEXLinuxTests/tests/CMakeLists.txt

```diff
@@ -8,6 +8,11 @@ unset(CMAKE_CXX_FLAGS)
 
 set(GENERATE_GUEST_INSTALL_TARGETS TRUE)
 
+# Use intel masm syntax. ATT style is maddening
+add_compile_options(-masm=intel)
+# Override a define so catch2 doesn't use ATT style inline asm
+add_definitions(-D"CATCH_BREAK_INTO_DEBUGGER\(\)"=)
```

Comment:
What's the effect of this? Sounds like this disables breakpoint triggering when a test failure is hit, which would be highly undesirable.

Reply:
This disabled the -b or `shouldDebugBreak` option. Changed it so it uses `__builtin_trap` so it still works.